Link to organizations page (#214)
OpenTermsArchive/opentermsarchive.org: README.md

# OpenTermsArchive.org

> This is the repository for `opentermsarchive.org`, the website that presents the [Open Terms Archive](https://github.com/OpenTermsArchive/) project.

Reviewer: ```suggestion
The website that presents Open Terms Archive to the general public.
```
